README.md

```diff
@@ -45,131 +56,180 @@ This environment variable will be used in the deployment steps.
+
+```bash
+ELB_CNAME="${DESIREDNAMESPACE}.${DOMAIN}"
+ELB_CERTIFICATE_ARN=$(aws acm list-certificates | \
+  jq '.CertificateSummaryList[] | select (.DomainName == "'${DOMAIN}'") | .CertificateArn')
+
+cat <<EOF > ingressvalues.yaml
+controller:
+  config:
+    ssl-redirect: "false"
+  scope:
+    enabled: true
+  publishService:
+    enabled: true
+  service:
+    targetPorts:
+      http: http
+      https: http
+    annotations:
+      external-dns.alpha.kubernetes.io/hostname: ${ELB_CNAME}
+      service.beta.kubernetes.io/aws-load-balancer-backend-protocol: "http"
+      service.beta.kubernetes.io/aws-load-balancer-connection-idle-timeout: '3600'
+      service.beta.kubernetes.io/aws-load-balancer-ssl-cert: ${ELB_CERTIFICATE_ARN}
+      service.beta.kubernetes.io/aws-load-balancer-ssl-ports: https
+EOF
+
+helm install stable/nginx-ingress --version=0.14.0 -f ingressvalues.yaml \
+  --namespace $DESIREDNAMESPACE
+
+```
+
+<!-- markdownlint-disable MD029 -->
+2. Get the nginx-ingress-controller release name from the previous command and set it as a varible:
+<!-- markdownlint-disable MD029 -->
+
+```bash
+export INGRESSRELEASE=knobby-wolf
+```
+
+<!-- markdownlint-disable MD029 -->
+3. Wait for the nginx-ingress-controller release to get deployed (When checking status your pod should be READY 1/1):
+<!-- markdownlint-enable MD029 -->
+
+```bash
+helm status $INGRESSRELEASE
+```
+
+<!-- markdownlint-disable MD029 -->
+4. Get the nginx-ingress-controller port for the infrastructure (NOTE! ONLY FOR MINIKUBE):
+<!-- markdownlint-enable MD029 -->
+
+```bash
+export INFRAPORT=$(kubectl get service $INGRESSRELEASE-nginx-ingress-controller --namespace $DESIREDNAMESPACE -o jsonpath='{.spec.ports[0].nodePort}')
+```
+
+<!-- markdownlint-disable MD029 -->
+5. Get Minikube or ELB IP and set it as a variable for future use:
+<!-- markdownlint-disable MD029 -->
+
+```bash
+#ON MINIKUBE
+export ELBADDRESS=$(minikube ip)
+
+#ON AWS
+export ELBADDRESS=$(kubectl get services $INGRESSRELEASE-nginx-ingress-controller --namespace=$DESIREDNAMESPACE -o jsonpath='{.status.loadBalancer.ingress[0].hostname}')
+```
+
+### 2. Deploy the keycloak charts
+
 ```bash
 helm repo add alfresco-incubator http://kubernetes-charts.alfresco.com/incubator
+
+#ON MINIKUBE
 helm install alfresco-incubator/alfresco-identity-service \
---set ingressHostName=$ELBADDRESS \
---namespace $DESIREDNAMESPACE
+  --set ingressHostName=$ELBADDRESS \
+  --namespace $DESIREDNAMESPACE
+
+#ON AWS
+
+helm install alfresco-incubator/alfresco-identity-service \
+  --set ingressHostName=$ELBADDRESS \
+  --namespace $DESIREDNAMESPACE
 ```
 
 ## Customizing the Realm
 
 ### Customizing the Realm During Deployment
 
-1. You will need a realm file. A [sample realm](./alfresco-realm.json) file is provided.
+1. You will need a realm json, similar to `alfresco-realm.json`
```

Comment:
Just a minor note: why don't we want to point users directly to the sample realm file?